lua/astrocommunity/completion/codeium-nvim/init.lua

```diff
@@ -36,28 +36,28 @@ return {
         end,
       },
     },
-    {
-      "hrsh7th/nvim-cmp",
-      optional = true,
-      dependencies = { "Exafunction/codeium.nvim" },
-      opts = function(_, opts)
-        -- Inject codeium into cmp sources, with high priority
-        table.insert(opts.sources, 1, {
-          name = "codeium",
-          group_index = 1,
-          priority = 10000,
-        })
-      end,
-    },
-    {
-      "onsails/lspkind.nvim",
-      optional = true,
-      -- Adds icon for codeium using lspkind
-      opts = function(_, opts)
-        opts.symbol_map = {
-          Codeium = require("astroui").get_icon("Codeium", 1, true),
-        }
-      end,
-    },
+  },
+  {
+    "hrsh7th/nvim-cmp",
+    optional = true,
+    dependencies = { "Exafunction/codeium.nvim" },
+    opts = function(_, opts)
+      -- Inject codeium into cmp sources, with high priority
+      table.insert(opts.sources, 1, {
+        name = "codeium",
+        group_index = 1,
+        priority = 10000,
+      })
+    end,
+  },
+  {
+    "onsails/lspkind.nvim",
+    optional = true,
+    -- Adds icon for codeium using lspkind
+    opts = function(_, opts)
+      opts.symbol_map = {
+        Codeium = require("astroui").get_icon("Codeium", 1, true),
+      }
```

Comment:
```suggestion
      if not opts.symbol_map then opts.symbol_map = {} end
      opts.symbol_map.Codeium = require("astroui").get_icon("Codeium", 1, true)
```

This is a bug and will overwrite `symbol_map` if it already exists